datasync/chaindatafetcher/kas/repository_token_transfer.go

```diff
@@ -0,0 +1,129 @@
+package kas
+
+import (
+	"fmt"
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"math/big"
+	"strings"
+)
+
+var tokenTransferEventHash = common.HexToHash("0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef")
+
+// splitToWords divides log data to the words.
+func splitToWords(data []byte) []common.Hash {
+	var words []common.Hash
```

Comment:
We need to check whether`len(data) % common.HashLength == 0`. 
If not, it will cause panic

Reply:
check logic is added. 5a41a8aabea1abca07527c77b39da4be05d8c2c8